lib/NativeCall.rakumod

```diff
@@ -260,6 +263,144 @@ sub resolve-libname($libname) {
     $libname.platform-library-name.Str
 }
 
+use MONKEY-SEE-NO-EVAL;
+my $use-dispatcher = so $*RAKU.compiler.?supports-op('dispatch_v') && EVAL q:to/CODE/;
```

Comment:
Could the `EVAL` be done at `BEGIN` time, so we don't have to do the parse/compile of this every time `NativeCall` is first used in a program?